test/CMakeLists.txt

```diff
@@ -1,28 +1,33 @@
-if (NOT TARGET gtest)
-  # gtest targets cmake version 2.6 which throws warnings for policy CMP0042 on
-  # newer cmakes. This sets the default global setting for that policy.
-  set(CMAKE_POLICY_DEFAULT_CMP0042 NEW)
```

Comment:
Are you sure this is not a problem any more ?

Reply:
I have not tested it, but I realized it is not a problem because you made cmake minimum version as 3.12.

Reply:
I mean, what does it do the range ? Does it add any special behavior is what I am asking ?

Reply:
My mistake for misleading reply. While testing with 3.16 (on mac) I did not get any policy warning, so I removed it.

Also, we have not received any warnings in travis logs, e.g.: https://travis-ci.org/github/9prady9/MeshIO/builds/663842224?utm_source=github_status&utm_medium=notification

Reply:
Lets leave this in for now. I will check git history when I added this and figure what causes this and remove it if not needed. If you want to do it. that is fine by me.

Reply:
I will retain this as part of this branch. It will take me sometime to understand whether it is really needed, so I shall do this later depending on time, while you also being doing this.

Also gtest does not target 2.6 now it made 2.8.8 as minimum,  so I am not sure if it is required. By taking a quick look at [CMP0042](https://cmake.org/cmake/help/v3.0/policy/CMP0042.html), I realized it might still need this, but travis builds and my local builds show otherwise.

Reply:
Retained this policy.